wooey/admin.py

```diff
@@ -68,8 +69,14 @@ def script_versions(self, obj):
         return ', '.join(['{}: {}'.format(script_version.script.script_name, script_version.script_iteration) for script_version in obj.script_version.all()])
 
 class ScriptVersionAdmin(ModelAdmin):
-    list_display = ('script', 'script_version', 'script_iteration', 'default_version')
-
+    list_display = ('script', 'script_version', 'script_iteration', 'default_version', 'created_by', 'modified_by')
+    readonly_fields = ('created_date', 'created_by', 'modified_date', 'modified_by')
+
+    def save_model(self, request, obj, form, change):
+        if not change:
```

Comment:
If this already exists won't we always update created by with the last user? Seems like this should be checking if created by is empty

Reply:
Nope! `change` is `True` only when updating an existing entry and `False` when creating a new one.
<img width="1256" alt="image" src="https://user-images.githubusercontent.com/1281521/183267211-8c813cea-dcba-4d0c-8c8a-f4b0b22f5e8a.png">